meta-genivi-dev/recipes-dev-platform/images/genivi-dev-platform.bb

```diff
@@ -32,6 +32,7 @@ IMAGE_INSTALL_append = " \
     packagegroup-gdp-qt5 \
     packagegroup-gdp-rvi \
     packagegroup-gdp-dev \
+    chromium-wayland \
```

Comment:
Without this change, wouldn't the first change be broken? A patch shouldn't depend on a following patch in the tree/series but only the other way around. Maybe this patch should just come first?